Source/options.h

```diff
@@ -573,7 +575,10 @@ struct KeymapperOptions : OptionCategoryBase {
 
 	void AddAction(
 	    string_view key, string_view name, string_view description, int defaultKey,
-	    std::function<void()> actionPressed, std::function<void()> actionReleased = nullptr, std::function<bool()> enable = nullptr, int index = -1);
+	    std::function<void()> actionPressed,
+	    std::function<void()> actionReleased = nullptr,
+	    std::function<bool()> enable = nullptr,
+	    size_t index = std::numeric_limits<size_t>::max());

```

Comment:
This is a non-zero value every time it's relevant, could use 0 as the default/unused marker.